js/webui.js

```diff
@@ -461,6 +462,7 @@ var theWebUI =
 			}
 			return sorter;
 		}
+		this.createSpeedGraph();
```

Comment:
Do we need to create the speed graph before the webui is initialized? If there's an error the entire page won't load.

Reply:
I simply tested this fix by confirming that no JS error occurred when inserting `theWebUI.systemInfo.rTorrent.started = true;`  here https://github.com/Novik/ruTorrent/blob/master/js/content.js#L677.

It appears to me that `theWebUI.resize()` https://github.com/Novik/ruTorrent/blob/d2ab9d725a0dc856a1f16020fe7b1a95b84863b5/js/webui.js#L2801 assumes here that the content of the tabs, including the speed tab, is initialized in `theWebUI.config()`. So, I think, `this.createSpeedGraph()` should also stay in `config()`. 

> It is true however, that webui.js is doing too much and should be split into smaller [JavaScript modules and/or classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#modules_and_classes) (for instance: into classes for peers,files,trackers, (speed) tabs).